apps/dapp-console-api/package.json

```diff
@@ -14,12 +14,16 @@
     "kill": "npx kill-port 7300",
     "build": "tsup && pnpm tsc",
     "start": "node build/cmd/run.js",
-    "docker:services:up": "docker compose up -d redis",
+    "docker:services:up": "docker compose up -d postgres redis",
```

Comment:
would also add a docker compose down command here to make developing easier so we can tear it down faster

Reply:
there is already a docker:down command. are you saying there should be another one that just brings down the services?

Reply:
it's probably fine to use the other one, what i'm wondering will happen is if someone is developing thats not as used to the backend code running the down command will bring down the db and wipe it which might be unexpected if you're thinking it's only bringing down the api server

the way the other one is setup allows you to pick what you want to bring down which is great and allows us to pick and choose and probably good for the rest of the team to continue to get used to compose more